apps/condo/domains/banking/utils/validate/routingNumber.utils.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Routing number validator
+ *
+ * The following checks are performed:
+ * 1) Сhecking for emptiness
+ * 2) Checking for length and format (Consists of 9 digits)
+ *
+ * For RU routing number:
+ * *** In Russia, number routing is equivalent to BIC***
+ * 1) For RU organizations country code (first two digits) is 04
+ *
+ * Example:
+ * RU - 046000987
+ */
+
+//GENERAL ERRORS
+const EMPTY = 'Routing number is empty'
+const NOT_NUMERIC = 'Routing number can contain only digits'
+const WRONG_LENGTH = 'Routing number length was expected to be 9, but received '
+
+//RU ERRORS
+const WRONG_RU_COUNTRY_CODE = 'For RU organizations country code is 04, but routing number have '
+
+
+class RoutingNumberValidation {
```

Comment:
Why you need a class here?
It gets used only in one place `validateRoutingNumber` and it does not even exported…

Reply:
Not using class right now